Minji/02_배열제어.py

```diff
@@ -0,0 +1,11 @@
+def solution(arr):
+	for i in range(0, len(arr)):
```

Comment:
시작 index가 0인 경우 아래와 같이 생략할 수 있습니다
```suggestion
	for i in range(len(arr)):
```